.github/workflows/setup_env.sh

```diff
@@ -70,6 +70,16 @@ if [ "${device}" = "gpu_amd" ]; then
     hipcc --version
 fi
 
+if [ "${device}" = "gpu_intel" ]; then
+    print "======================================== Load Intel oneAPI"
+    export gpu_backend=onemkl
+    module unload intel-mkl
+    module load intel-oneapi-compilers
+    module load intel-oneapi-mkl
```

Comment:
Please add `quiet` on `module load`. It greatly reduces the output in the log.

We could use just load `intel-oneapi-mkl` above for all cases, and never load the older `intel-mkl` package.

Reply:
OK, I've replaced intel-mkl with oneapi version and added quiet to the remaining "module load".